todoman/model.py

```diff
@@ -587,6 +589,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
+            extra_where.append('AND priority = ' + priority)
```

Comment:
@untitaker suggested that `--priority medium` would be the way to filter and then medium would be converted to the value 5 using the `validate_priority_param`.
Do we want to keep filtering using integer values as well?